pypuf/simulation.py

```diff
@@ -106,3 +106,36 @@ def ltf_eval(self, inputs):
                 for l in range(self.k)
             ])
         )
+
+
+class NoisyLTFArray(LTFArray):
+    """
+    Class that simulates k LTFs with n bits and a constant term each 
+    with noise effect and constant bias (=scale) added.
```

Comment:
sigma_noise = scale of overall weights * noise     , while
bias = scale of one weight